client/reader/following-manage/index.jsx

```diff
@@ -17,7 +17,7 @@ import DocumentHead from 'components/data/document-head';
 import SearchInput from 'components/search';
 import HeaderCake from 'components/header-cake';
 import ReaderMain from 'reader/components/reader-main';
-import getBlockedSites from 'state/selectors/get-blocked-sites';
+import getBlockedSites from 'state/reader/site-blocks/selectors/get-blocked-sites';
```

Comment:
I'd prefer to do my imports as:
```js
import { getBlockedSites } from 'state/reader/site-blocks/selectors';
```
i.e., from a single umbrella `*/selectors` module.

Reply:
Good idea! 👍 I added an index file that re-exports all the selectors, and updated all the imports to use that instead.